### PR TITLE
feature/166468405_add_permissions_screen

### PR DESCRIPTION
**Pull request by vyvil**
This PR includes changes on Android version of the app.
Changes:
1. Added permissions screen to list and explain permissions required by the app as well as the data that is being collected by the app and third parties.
2. Removed unnecessary Android permissions.

- android.permission.RECEIVE_BOOT_COMPLETED
- android.permission.MANAGE_DOCUMENTS
- android.permission.ACCESS_COARSE_LOCATION (had to be removed by adding `tools:node="remove"` as it gets appended to merged manifest)
- android.permission.ACCESS_FINE_LOCATION (had to be removed by adding `tools:node="remove"` as it gets appended to merged manifest)
- android.permission.READ_CONTACTS (legacy of Expo)
- android.permission.READ_CALENDAR (legacy of Expo)
- android.permission.WRITE_CALENDAR (legacy of Expo)
- android.permission.WRITE_SETTINGS (legacy of Expo)
- android.permission.USE_FINGERPRINT (was included twice)

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,34 +12,28 @@
...
-    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
 
     <!-- ADD PERMISSIONS HERE -->
     <!-- BEGIN OPTIONAL PERMISSIONS -->
-    <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
     <uses-permission android:name="android.permission.READ_INTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
-    <uses-permission android:name="android.permission.USE_FINGERPRINT" />
```
Comment (jsidorenko) [inline android/app/src/main/AndroidManifest.xml]: Don't we need this permission for biometrics?

Comment (alazarevski) [reply android/app/src/main/AndroidManifest.xml]: it's duplicated from line 40

Comment (vyvil) [reply android/app/src/main/AndroidManifest.xml]: It was added twice :)

Comment (jsidorenko) [reply android/app/src/main/AndroidManifest.xml]: 👍

**Pull request merged**